src/box/sql.c

```diff
@@ -367,7 +367,16 @@ sql_ephemeral_space_new(const struct sql_space_info *info)
 		parts[i].exclude_null = false;
```

Comment:
In the commit about "DISTINCT" you said there was a segfault before, but without your fix the test fails with a Lua error, not a crash. Shouldn't you add some data to the tested spaces?

Reply:
True, thank you. Fixed.